### PR TITLE
CSI 0.3 has been removed in K8s 1.17

### PR DESCRIPTION
**Pull request by msau42**

### CODE REVIEW

File: book/src/kubernetes-changelog.md [hunk 1]
```diff
@@ -27,19 +37,19 @@ details on individual features, visit the [Features section](features.md).
...
-* New alpha features:
+* Alpha
     * Volume expansion
-* New `storage.k8s.io/v1beta1` `CSINode` and `CSIDriver` objects were introduced.
+* `storage.k8s.io/v1beta1` `CSINode` and `CSIDriver` objects introduced.
 
 ## Kubernetes 1.13
 
 ### Deprecations
-* CSI spec 0.2 and 0.3 are deprecated and support will be removed in Kubernetes 1.18.
```
Comment (jsafrane) [inline book/src/kubernetes-changelog.md]: hmmm :-)

Comment (msau42) [reply book/src/kubernetes-changelog.md]: yeah that was my mistake... the k8s release notes actually said removal in 1.16...

**Pull request merged**